NuRadioReco/framework/base_trace.py

```diff
@@ -37,6 +37,35 @@ def get_trace(self):
             self._frequency_spectrum = None
         return self._time_trace
 
+    def get_filtered_trace(self, passband, filter_type='butter', order=10):
```

Comment:
can you use the `get_filter` method of the channelBandpassFilter module instead? Then we only need to implement filters at one common place. The parameters `station_id`, `channel_id`, `det` can just be `None`

Reply:
Done, while I was at it I also implemented this for the E-field filters. I decided to define the filters in an extra function outside the channelBandPassFilter class. Otherwise I would have had to create a channelBandPassFilter object every time I want to filter a trace and that would probably be pretty bad for performance

Reply:
Oops, I accidentally create a loop in the imports. The filters are now in utilities. That should fix it